src/lib.rs

```diff
@@ -333,259 +330,12 @@ pub fn handle_force_redraws(app: &mut App) {
-            if proc_widget_state.is_process_sort_descending {
-                to_sort_vec.reverse();
-            }
-        }
-        ProcessSorting::Pid => {
-            if !proc_widget_state.is_grouped {
-                to_sort_vec.sort_by(|a, b| {
-                    utils::gen_util::get_ordering(
-                        a.pid,
-                        b.pid,
-                        proc_widget_state.is_process_sort_descending,
-                    )
-                });
-            }
-        }
-        ProcessSorting::ReadPerSecond => {
-            to_sort_vec.sort_by(|a, b| {
-                utils::gen_util::get_ordering(
-                    a.rps_f64,
-                    b.rps_f64,
-                    proc_widget_state.is_process_sort_descending,
-                )
-            });
-        }
-        ProcessSorting::WritePerSecond => {
-            to_sort_vec.sort_by(|a, b| {
-                utils::gen_util::get_ordering(
-                    a.wps_f64,
-                    b.wps_f64,
-                    proc_widget_state.is_process_sort_descending,
-                )
-            });
-        }
-        ProcessSorting::TotalRead => {
-            to_sort_vec.sort_by(|a, b| {
-                utils::gen_util::get_ordering(
-                    a.tr_f64,
-                    b.tr_f64,
-                    proc_widget_state.is_process_sort_descending,
-                )
-            });
-        }
-        ProcessSorting::TotalWrite => {
-            to_sort_vec.sort_by(|a, b| {
-                utils::gen_util::get_ordering(
-                    a.tw_f64,
-                    b.tw_f64,
-                    proc_widget_state.is_process_sort_descending,
-                )
-            });
-        }
-        ProcessSorting::State => {
-            to_sort_vec.sort_by_cached_key(|c| c.process_state.to_lowercase());
-            if proc_widget_state.is_process_sort_descending {
-                to_sort_vec.reverse();
-            }
-        }
-        ProcessSorting::User => to_sort_vec.sort_by(|a, b| match (&a.user, &b.user) {
-            (Some(user_a), Some(user_b)) => utils::gen_util::get_ordering(
-                user_a.to_lowercase(),
-                user_b.to_lowercase(),
-                proc_widget_state.is_process_sort_descending,
-            ),
-            (Some(_), None) => std::cmp::Ordering::Less,
-            (None, Some(_)) => std::cmp::Ordering::Greater,
-            (None, None) => std::cmp::Ordering::Less,
-        }),
-        ProcessSorting::Count => {
-            if proc_widget_state.is_grouped {
-                to_sort_vec.sort_by(|a, b| {
-                    utils::gen_util::get_ordering(
-                        a.group_pids.len(),
-                        b.group_pids.len(),
-                        proc_widget_state.is_process_sort_descending,
-                    )
-                });
-            }
-        }
-    }
-}
-
```

Comment:
Also all removed in favour of moving the logic into `ProcWidget`, and with slightly more updated logic.